templates/website/carousel/controls/_controls.html.twig

```diff
@@ -0,0 +1,10 @@
+<div data-glide-el="controls[nav]">
+    <button class="arrow-left arrow" data-glide-dir="<">&lt;</button>
+    <button class="arrow-right arrow" data-glide-dir=">">&gt;</button>
+</div>
+<div class="glide__bullets" data-glide-el="controls[nav]">
+    <!--elementy generowane-->
```

Comment:
Get rid of these comments. if they must be there, let's leave them in English, but first consider whether they are really necessary